custom_components/rpi_waveshare_ups/__init__.py

```diff
@@ -41,10 +42,12 @@ def __exit__(self, exc_type, exc, traceback) -> None:
         """Exit magic method."""
         self._close()
 
-    def __init__(self, i2c_bus: int, i2c_address: int) -> None:
+    def __init__(self, i2c_bus: int, i2c_address: int, is_model_d: bool) -> None:
```

Comment:
I wonder if this might be more future proof if the model is passed in and compared against rather than using `is_model_d`.  That way the class could accommodate any model.  Maybe that isn't for this PR though and I can handle that afterwards.

Reply:
Sounds good and feel free to update.